pkg/api/pss_test.go

```diff
@@ -15,29 +17,31 @@ import (
 	"testing"
 	"time"
 
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/ethersphere/bee/pkg/crypto"
 	"github.com/ethersphere/bee/pkg/jsonhttp"
 	"github.com/ethersphere/bee/pkg/jsonhttp/jsonhttptest"
 	"github.com/ethersphere/bee/pkg/logging"
 	"github.com/ethersphere/bee/pkg/pss"
 	"github.com/ethersphere/bee/pkg/pushsync"
 	"github.com/ethersphere/bee/pkg/storage/mock"
 	"github.com/ethersphere/bee/pkg/swarm"
-	"github.com/ethersphere/bee/pkg/trojan"
 	"github.com/gorilla/websocket"
 )
 
 var (
-	target  = trojan.Target([]byte{1})
-	targets = trojan.Targets([]trojan.Target{target})
+	target  = pss.Target([]byte{1})
+	targets = pss.Targets([]pss.Target{target})
```

Comment:
if you are already refactoring these things between packages, i would suggest to completely remove `Targets` type. It does not serve any purpose, we don't need type alias to represent a slice. it also makes the code clunky in many places

Reply:
@zelig do you have suggestion for this, since you modified packages ?

Reply:
yes, but ultimately we want to support bit-level variable length targets but i dont mind\